src/components/pages/FeatureRequestPage.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
```

Comment:
Need to remove the import of `useState` here, as we're not using it. Check below imports to see what else we're not using, the linter should automatically catch them.